### PR TITLE
minor cosmetics + use dedicated dir for chain dumps

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

File: polkadot-parachains/src/chain_spec.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@ pub fn shell_chain_spec(id: ParaId, genesis_keys: GenesisKeys, relay_chain: Rela
 		GenesisKeys::WellKnown => (WellKnownKeys::root(), WellKnownKeys::endowed(), WellKnownKeys::authorities()),
 	};
 
-	let chain_name = format!("IntegriTEE Shell{}", get_chain_name_ext(&relay_chain.chain_type()));
+	let chain_name = format!("IntegriTEE Shell{}", relay_chain.name_ext());
```
Comment (clangenb) [inline polkadot-parachains/src/chain_spec.rs:104]: @brenzi I have seen you removed the chain name extensions in master. I suggest we leave it here to disambiguate chains in polkadot-js app. With `ChainType::Live` we append nothing anyhow.

Comment (brenzi) [reply polkadot-parachains/src/chain_spec.rs:104]: that's exactly the point. I don't want to enter 4 configurations in js/apps. The problem, however is that apps doesn't recognize any of my attempts to set color....I might revert this if no success

Comment (clangenb) [reply polkadot-parachains/src/chain_spec.rs:104]: Ok, got it. I will remove it from this pr then.

**Comment by clangenb** [comment]
I see you reopened this, how do you want to proceed with the chain-name extenesions?

File: polkadot-parachains/src/chain_spec.rs (hunk 1 above)
Comment (brenzi) [reply polkadot-parachains/src/chain_spec.rs:104]: why is there still a diff here? can we make this PR only about dump dir?

**Comment by brenzi** [comment]
plz rebase and ensure it's only about dump dir. then we can merge

File: polkadot-parachains/src/chain_spec.rs (hunk 1 above)
Comment (clangenb) [reply polkadot-parachains/src/chain_spec.rs:104]: The dump-dir part is already merged. The question is whether you want to have chan_spec-specific display names.

* If yes, I will rebase and we merge the chain_name cosmetics stuff.
* If not, we can close this pr.

**Pull request closed without merging**